addons/mail/sfu_server/src/models/session.js

```diff
@@ -0,0 +1,570 @@
+                webRtcServer: this._channel.webRtcServer,
+            });
+            this.once("close", () => {
+                this._ctsTransport?.close();
+                this._stcTransport?.close();
+            });
+            this._clientCapabilities = await this.bus.request({
+                name: SERVER_REQUEST.INIT_TRANSPORTS,
+                payload: {
+                    capabilities: this._channel.router.rtpCapabilities,
+                    /** @type{transportConfig} */
+                    stcConfig: {
+                        id: this._stcTransport.id,
+                        iceParameters: this._stcTransport.iceParameters,
+                        iceCandidates: this._stcTransport.iceCandidates,
+                        dtlsParameters: this._stcTransport.dtlsParameters,
+                        sctpParameters: this._stcTransport.sctpParameters,
+                    },
+                    /** @type{transportConfig} */
+                    ctsConfig: {
+                        id: this._ctsTransport.id,
+                        iceParameters: this._ctsTransport.iceParameters,
+                        iceCandidates: this._ctsTransport.iceCandidates,
+                        dtlsParameters: this._ctsTransport.dtlsParameters,
+                        sctpParameters: this._ctsTransport.sctpParameters,
+                    },
+                },
+            });
+            await this._ctsTransport.setMaxIncomingBitrate(config.MAX_BITRATE_IN);
+            await this._stcTransport.setMaxOutgoingBitrate(config.MAX_BITRATE_OUT);
+        } catch (error) {
+            logger.error(
+                `[${this.identifier}] failed to create transports (${error.message}), closing session`
+            );
+            this.close(SESSION_CLOSE_CODE.ERROR, error.message);
+            return;
+        }
+        /**
+         * The session is considered connected as soon as the transports are ready,
+         * regardless of whether we are producing or consuming streams, as this can
+         * happen arbitrarily late in the lifecycle of a session.
+         */
+        this.state = SESSION_STATE.CONNECTED;
+        logger.info(`[${this.identifier}] connected`);
+
+        // starting streams
+        const promises = [];
+        for (const session of this._channel.sessions.values()) {
+            promises.push(this.consume(session));
+            promises.push(session.consume(this));
+        }
+        await Promise.all(promises);
+        logger.info(`[${this.identifier}] streaming`);
+    }
+
+    /**
+     * Creates missing consumers for each producer of `params.session` and sets their appropriate `paused` state.
+     * This batches the consumption of all streams.
+     *
+     * @param {Session} session
+     */
+    async consume(session) {
+        if (this === session) {
+            return;
+        }
+        if (session.state !== SESSION_STATE.CONNECTED) {
+            return;
+        }
```

Comment:
It should be fine, It can happen as we call a `consume()` on the other sessions of the channel when the current session update its tracks, but there are some sessions that may still be in an initialization phase when that happens.

If that is the case, it will prevent them from trying to make consumers when they can't (no transports), and as soon as they reach the `connected` state, they will start the consumptions that are missing themselves.